### PR TITLE
Update Rust option in the Implementations list

### PR DESCRIPTION
**Pull request by jovfer**
Replace old Rust implementation reference (draft v2) with the latest one in OWF.

### CODE REVIEW

**Review by bc-pi — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]

**Comment by Sakurann** [comment]
just reached out to @kushaldas to make sure he is aware of this, just to be sure.

**Review by Sakurann — CHANGES_REQUESTED** [review]
just until there is a confirmation there is a relationship between this new URL and kushal's one

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The artifacts generated for the examples (e.g., serialized SD-JWTs, Disclosures,
  * Kotlin: [SD-JWT-Kotlin (ID Union)](https://github.com/IDunion/SD-JWT-Kotlin)
  * Kotlin: [eudi-lib-jvm-sdjwt (EU Digital Identity Wallet)](https://github.com/eu-digital-identity-wallet/eudi-lib-jvm-sdjwt-kt)
  * Swift: [eudi-lib-sdjwt-swift (EU Digital Identity Wallet)](https://github.com/eu-digital-identity-wallet/eudi-lib-sdjwt-swift)
- * Rust: [sd_jwt](https://github.com/kushaldas/sd_jwt)
+ * Rust: [sd-jwt-rs (OWF)](https://github.com/openwallet-foundation-labs/sd-jwt-rust)
```
Comment (Sakurann) [inline README.md:47]:
```suggestion
 * Rust: [sd_jwt](https://github.com/kushaldas/sd_jwt)
 * Rust: [sd-jwt-rs (OWF)](https://github.com/openwallet-foundation-labs/sd-jwt-rust)
```

**Review by Sakurann — CHANGES_REQUESTED** [review]
confirmed that the current proposed change was replacing a library by another person, so please add a new line instead of replacing another repo

**Review by bc-pi — CHANGES_REQUESTED** [review]
> confirmed that the current proposed change was replacing a library by another person, so please add a new line instead of replacing another repo

please fix and also explain the intent

**Comment by danielfett** [comment]
@jovfer Can you please address the comments/proposed changes above?

**Comment by jovfer** [comment]
Updated. I'm unsure if it's worth keeping the link to the outdated implementation, but it's probably out of scope for this PR.

**Review by bc-pi — APPROVED** [review]

**Comment by Sakurann** [comment]
> Updated. I'm unsure if it's worth keeping the link to the outdated implementation, but it's probably out of scope for this PR.

thank you for updating. please do not modify the repositories of other people.

**Pull request merged**